example/trino_cluster/main.tf

```diff
@@ -57,15 +57,31 @@ module "presto" {
 
   # Vault provided credentials
   minio_vault_secret = {
-    use_vault_provider       = true
-    vault_kv_policy_name     = "kv-secret"
-    vault_kv_path            = "secret/data/dev/minio"
-    vault_kv_access_key_name = "access_key"
-    vault_kv_secret_key_name = "secret_key"
+    use_vault_provider         = true
+    vault_kv_policy_name       = "kv-secret"
+    vault_kv_path              = "secret/data/dev/minio"
+    vault_kv_field_access_name = "access_key"
+    vault_kv_field_secret_name = "secret_key"
   }
 
+  postgres_service = {
+    service_name  = module.postgres.service_name
+    port          = module.postgres.port
+    username      = module.postgres.username
+    password      = module.postgres.password
+    database_name = module.postgres.database_name
+  }
+  postgres_vault_secret = {
+    use_vault_provider      = false
+    vault_kv_policy_name    = ""
+    vault_kv_path           = ""
+    vault_kv_field_username = ""
+    vault_kv_field_password = ""
+  }
 }
 
+# TODO: Update dependent modules version
```

Comment:
What exactly does it mean? Update which dependent modules? Update variables?

Reply:
Dependent modules are postgres, minio etc. But I won't fix the trino_cluster in this version. I'll create a new issue regarding this since I believe it is a bigger task.

Reply:
#115